frontend/src/components/episode/episode.tsx

```diff
@@ -115,7 +115,12 @@ const Episode: React.FC = () => {
                 )}
                 <h1 className={styles.title}>{episode.name}</h1>
                 <p className={styles.description}>{episode.description}</p>
-                <p className={styles.status}>Status: {episode.status}</p>
+                <dl>
+                  <dt>Type: </dt>
+                  <dd>{episode.type}</dd>
+                  <dt>Status:</dt>
+                  <dd>{episode.status}</dd>
+                </dl>
```

Comment:
```suggestion
                <dl>
                   <div>
                                     <dt>Type: </dt>
                  <dd>{episode.type}</dd>
                   </div>
                   <div>
                                     <dt>Status:</dt>
                  <dd>{episode.status}</dd>
                   </div>
                </dl>
```
add style by classes